datareservoirio/authenticate.py

```diff
@@ -89,6 +91,19 @@ def __init__(self, client, auth_force=False, **kwargs):
             **kwargs,
         )
 
+        # Attention: Be careful when extending the list of retry_status!
+        retry_status = frozenset([413, 429, 502, 503, 504])
+        allowed_methods = frozenset(["GET", "POST", "PUT", "PATCH", "DELETE"])
+
+        persist = Retry(
+            total=3,
+            backoff_factor=0.5,
+            allowed_methods=allowed_methods,
+            status_forcelist=retry_status,
+            raise_on_status=False,
+        )
+        self.mount(environment.api_base_url, HTTPAdapter(max_retries=persist))
```

Comment:
Not to self: Check out which URL is the api_base_url, is this hooking the retry up to all calls or only a subset of them?
And what does this max_retries=persist means? Is this the way of hooking the retry policy into the pipeline somehow?

Reply:
`max_retries` -> `Retry` class to define the retry policy.

Reply:
Good, I just could not find the docs...